nightshift/datastore_transactions.py

```diff
@@ -3,6 +3,38 @@
 from nightshift.datastore import Resource
 
 
+def init_db():
+    """
+    Initiates the database and prepopulates needed tables
+
+    Args:
+        session:                sqlalchemy.Session instance
+    """
+    from sqlalchemy import create_engine
+
+    from .constants import LIBRARIES, RESOURCE_CATEGORIES
+    from .datastore import Base, Library, ResourceCategory, DataAccessLayer
```

Comment:
Just a stylistic thing, but unless necessary I like to keep imports at the top of each file so that I always know where to look for them

Reply:
Done.